<launch>
    <rosparam command="load"
        file="$(find split0914_hrm)/launch/buddy/master/config/control.yaml" />

    <node pkg="videoray" type="pro4_node" name="pro4_driver">
  	<remap from="pwm_in" to="pwm_out" />
	<param name="PortName" value="/dev/ttyUSB1" />
	<param name="Baud" value="115200" />
        <param name="max" value="1.0"/>
        <param name="min" value="-1.0"/>
    </node>
    <node pkg="pladypos" name="pro4_node_v1" type="pro4_node_v1">
      <param name="maxThrust" value="1"/>
      <param name="minThrust" value="-1"/>
      <param name="max_cap" value="1.0"/>
      <remap from="tauIn" to="tauOut" />
      <!-- <remap from="tauAch" to="tauAchPl" /> -->
    </node>

    <node
        pkg="labust_control"
        type="velocity_control"
        name="velcon">
                <remap from="nuRef" to="nuRefMerged" />
    </node>

     <node pkg="joy" name="joystick" type="joy_node">
                <param name="dev" value="/dev/input/js1" />
                <param name="coalesce_interval" value="0.01" />
                <param name="autorepeat_rate" value="10" />
                <param name="deadzone" value="0.05" />
        </node>


</launch>
